src/index.js

```diff
@@ -0,0 +1,743 @@
+   */
+  shareCollection({ id, ...params } = {}) {
+    if (!id) {
+      return rejectValidation('collection', 'id');
+    }
+    if (!params.recipients) {
+      return rejectValidation('collection', 'recipients');
+    }
+    if (!params.collectionOptions) {
+      return rejectValidation('collection', 'collectionOptions');
+    }
+
+    return this.api.send('POST', `api/v4/collections/${id}/share/`, params);
+  }
+
+  /**
+   * Get a list of brands and subbrands
+   * @see {@link https://bynder.docs.apiary.io/#reference/security-roles/specific-security-profile/retrieve-brands-and-subbrands}
+   * @return {Promise}
+   */
+  getBrands() {
+    return this.api.send('GET', 'api/v4/brands/');
+  }
+
+  /**
+   * Get options for a metaproperty
+   * @see {@link https://bynder.docs.apiary.io/#reference/metaproperties/metaproperty-option-operations/retrieve-metaproperty-options|API Call}
+   * @param {Object} params={} - An object containing the id of the desired metaproperty and parameters accepted by the
+   * API to narrow the query for options
+   * @param {String} params.id - The id of the desired metaproperty.
+   * @return {Promise<array>} Response - Returns a Promise that, when fulfilled, will either return an Array with the
+   * metaproperties or an Error with the problem.
+   */
+  getMetapropertyOptions({ id, ...params } = {}) {
+    if (!id) {
+      return rejectValidation('metapropertyOption', 'id');
+    }
+
+    return this.api.send('GET', `api/v4/metaproperties/${id}/options/`, params);
+  }
+
+  /**
+   * Get the download url of original file.
+   * @see {@link https://bynder.docs.apiary.io/#reference/download/download-operations/retrieve-asset-download-location|API Call}
+   * @param {Object} params - An object containing the parameters accepted by the API to change in the asset.
+   * @param {String} params.id - The id of the desired asset.
+   * @returns {Promise}
+   */
+  getMediaDownloadUrl({ id, ...params } = {}) {
+    if (!id) {
+      return rejectValidation('media', 'id');
+    }
+
+    return this.api.send('GET', `api/v4/media/${id}/download`, { ...params });
+  }
+
+  /**
+   * Finalises the file upload when all chunks finished uploading and registers it in Bynder.
+   * @see {@link https://bynder.docs.apiary.io/#reference/upload-assets/4-finalise-a-completely-uploaded-file/finalise-a-completely-uploaded-file}
+   * @access private
+   * @param {String} fileId Unique file identifier
+   * @param {String} fileName Original file name
+   * @param {Number} chunksCount Number of chunks
+   * @param {Number} fileSize File byte size
+   * @return {Promise<object>} Object containing the correlation ID (`correlationId`) and file ID (`fileId`) of the upload
+   */
+  _finaliseUpload(fileId, fileName, chunksCount, fileSize) {
+    return this.api.send('POST', `v7/file_cmds/upload/${fileId}/finalise_api`, {
+      chunksCount, fileName, fileSize,
+      intent: DEFAULT_UPLOAD_INTENT,
```

Comment:
This can be left out, since it will fallback to the correct one for the SDKs (which is not `upload_main_uploader_asset`, that one is for the UI uploader)

Reply:
Fixed.